packages/react-components/react-combobox/src/components/Combobox/useComboboxStyles.ts

```diff
@@ -130,17 +140,17 @@ const useInputStyles = makeStyles({
   small: {
     fontSize: tokens.fontSizeBase200,
     lineHeight: tokens.lineHeightBase200,
-    ...shorthands.padding('3px', 0, '3px', tokens.spacingHorizontalSNudge),
+    ...shorthands.padding('3px', 0, '3px', `calc(${tokens.spacingHorizontalSNudge} + ${tokens.spacingHorizontalXXS})`),
   },
   medium: {
     fontSize: tokens.fontSizeBase300,
     lineHeight: tokens.lineHeightBase300,
-    ...shorthands.padding('5px', 0, '5px', tokens.spacingHorizontalMNudge),
+    ...shorthands.padding('5px', 0, '5px', `calc(${tokens.spacingHorizontalMNudge} + ${tokens.spacingHorizontalXXS})`),
   },
   large: {
     fontSize: tokens.fontSizeBase400,
     lineHeight: tokens.lineHeightBase400,
-    ...shorthands.padding('7px', 0, '7px', tokens.spacingHorizontalM),
+    ...shorthands.padding('7px', 0, '7px', `calc(${tokens.spacingHorizontalM} + ${tokens.spacingHorizontalSNudge})`),
```

Comment:
is there a reason why this padding is horizontalM and HorizontalSNudge? Do we need a new token to define these new widths? Or maybe we need a comment to explain why these calculations are happening.

Reply:
we talked a while ago about using `calc` instead of adding an extra node within Dropdown to wrap the text, which would be the other option.

For combobox, it's pretty much just necessary, since `<input>`s can't have children